pkgs/development/libraries/flatpak/fix-test-paths.patch

```diff
@@ -35,21 +26,18 @@
      if test -n "${TEST_SKIP_CLEANUP:-}"; then
 --- a/tests/make-test-runtime.sh
 +++ b/tests/make-test-runtime.sh
-@@ -26,6 +26,7 @@
+@@ -26,6 +26,7 @@ EOF
  PATH="$PATH:/usr/sbin:/sbin"
  
  # Add bash and dependencies
 +mkdir -p ${DIR}/nix/store
  mkdir -p ${DIR}/usr/bin
  mkdir -p ${DIR}/usr/lib
  ln -s ../lib ${DIR}/usr/lib64
-@@ -35,73 +36,27 @@
- else
-     cp `which ldconfig` ${DIR}/usr/bin
- fi
--LIBS=`mktemp`
--BINS=`mktemp`
```

Comment:
Hmm, this probably shouldn't have been deleted though, when I look at it again. Not a big deal, but they're unused so.

Reply:
Pushed commit to add back these deletions.